SpatialGDK/Source/Public/Interop/SpatialTypebindingManager.h

```diff
@@ -55,17 +66,18 @@ struct FClassInfo
 {
 	GENERATED_BODY()
 
-	TMap<ERPCType, TArray<UFunction*>> RPCs;
+	UClass* Class;
+
+	TMap<EComponentType, TArray<UFunction*>> RPCs;
 	TMap<UFunction*, FRPCInfo> RPCInfoMap;
 
 	TArray<FHandoverPropertyInfo> HandoverProperties;
 
-	Worker_ComponentId SingleClientComponent;
-	Worker_ComponentId MultiClientComponent;
-	Worker_ComponentId HandoverComponent;
-	Worker_ComponentId RPCComponents[RPC_Count];
+	Worker_ComponentId SchemaComponents[EComponentType::TYPE_Count] = {};
```

Comment:
We don't do default initialization in class/structs. Add a constructor.

Reply:
Who said we don't? Whaaaaaaaaaaaaaaaaaaaaaat?

Reply:
So I went looking for it in our docs, but only found that Unreal explicitly say you can use it, with judgement. I remember a conversation where we said we didn't however, but perhaps we should bring it up again.

Reply:
I think in this case we should use it. This struct is essentially being used as a POD and I think it's clear enough what's happening.